nestkernel/connection_manager.cpp

```diff
@@ -464,6 +464,13 @@ nest::ConnectionManager::connect( TokenArray sources, TokenArray targets, const
 void
 nest::ConnectionManager::update_delay_extrema_()
 {
+  if ( kernel().simulation_manager.has_been_simulated() )
+  {
+    // Once simulation has started, min/max_delay can no longer change,
+    // so there is nothing to update.
+    return;
+  }
```

Comment:
@heplesser In `ConnectionManager::get_connections()` there is a call to `update_delay_extrema_()` if `connections_have_changed`. It seems like it is used for setting the secondary event data size. I'm not sure if your change affect this, so I just wanted to check if you are aware and have taken that into consideration with the change?

Reply:
Thanks for pointing this out, but this is safe:  `update_delay_extrema()`  is called at least once while `has_been_simulated()` is still false, so we will get the correct min/max values.